Jenkinsfile

```
@@ -7,110 +7,113 @@ pipeline {
     }
 
     stages {
-        stage('Build') {
-            parallel {
-                stage('Windows') {
-                    agent {
-                        label "docker-windows"
+        stage('docker-agent') {
+            matrix {
```

```suggestion
            failFast true
            matrix {
```
Tested in https://ci.jenkins.io/job/Packaging/job/docker-agent/job/PR-445/7/console with a replay and a '2022' Windows version failing as missing in this PR, the build stopped without waiting the completion of every branch.

---

Proposed in https://github.com/jenkinsci/docker-agent/pull/446